lib/ipfs/RequestHandler.ts

```diff
@@ -18,13 +18,23 @@ export default class RequestHandler {
 
   private serviceInfo: ServiceInfo;
 
+  /**
+   * initializes an instance of request handler
+   * @param fetchTimeoutInSeconds Timeout for fetch request. Fetch request will return `not-found` when timed-out.
+   * @param repo Optional IPFS datastore implementation.
+   */
+  public static async initialize (fetchTimeoutInSeconds: number, repo?: any) {
```

Comment:
Isn't ````create```` a better name for this function? 

Also, the ion repo needs to call this new function now right?

Reply:
Yes ION will have to call this instead of instantiation. Is create the verb? I see both create and initialize in the code base. Not sure if we designate them to certain behaviors.

Reply:
In my mind, ````initialize```` is for when the objected is already 'created' and needs to be 'initialized'. Every call to ````initialize```` is like calling a function on an existing object. Whereas ````create```` returns a new instance of the object which the caller needs to hold and dispose off as needed. Every call to ````create```` returns a new instance of an object (which is what we are doing here.

Another thought actually, looks like the constructor doesn't need to be public right? Small nit though so up to you.